### PR TITLE
Upgrade Gradle Wrapper to 3.3

### PR DESCRIPTION
**Pull request by szpak**
Also removed the wrapper section in build.gradle.
An upgrade is with just a command line switch, e.g.:
./gradlew wrapper --gradle-version=3.4-rc-1

It seems to be ~10% faster on my machine. On Travis it varies...

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -76,10 +76,6 @@ apply from: 'gradle/mockito-core/javadoc.gradle'
 apply from: 'gradle/mockito-core/license.gradle'
 apply from: 'gradle/mockito-core/testing.gradle'
 
-task wrapper(type: Wrapper) {
```
Comment (TimvdLippe) [inline build.gradle]: Maybe add the replacing command here in a comment or in the readme.

Comment (szpak) [inline build.gradle]: Done

**Comment by TimvdLippe** [comment]
Seems like Travis dislikes these changes :(

**Comment by szpak** [comment]
> ThreadsRunAllTestsHalfManualTest > shouldRunInMultipleThreads FAILED

Seems to be not related.

**Comment by szpak** [comment]
But the problem with that test exists...

**Pull request merged**